### PR TITLE
Show log messages in egui toast notifications

### PR DESCRIPTION
**Pull request by emilk**
Show INFO, WARN, and ERROR in GUI as toast notifications. Otherwise users can easily miss important messages.

I've wanted to do this for a loooong time.

TODO: check if we both log and show notifications in some cases

### Checklist
* [x] I have read and agree to [Contributor Guide](https://github.com/rerun-io/rerun/blob/main/CONTRIBUTING.md) and the [Code of Conduct](https://github.com/rerun-io/rerun/blob/main/CODE_OF_CONDUCT.md)

### CODE REVIEW

File: crates/re_log/src/channel_logger.rs [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+                tx: tx.into(),
+            },
+            rx,
+        )
+    }
+}
+
+impl log::Log for ChannelLogger {
+    fn enabled(&self, metadata: &log::Metadata<'_>) -> bool {
+        if metadata.target().starts_with("wgpu") || metadata.target().starts_with("naga") {
```
Comment (Wumpf) [inline crates/re_log/src/channel_logger.rs:29]: feels like we have this hardcoded in too many places now :/
Can we have the `LOUD_CRATES` logic/"database" move to a central place?

**Review by Wumpf — APPROVED** [review]
Tried spammy message with newlines. Kinda works out well enough.
image

File: crates/re_log/src/channel_logger.rs (hunk 1 above)
Comment (emilk) [reply crates/re_log/src/channel_logger.rs:29]: Or I spend a couple of hours fixing wgpu/naga…

**Review by Wumpf — COMMENTED** [review]
much nicer silencing levels, thanks!!

**Pull request merged**